plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -2846,21 +2822,18 @@ class AzureDscpConfiguration(AzureResource):
     source_port_ranges: Optional[List[AzureQosPortRange]] = field(default=None, metadata={'description': 'Sources port ranges.'})  # fmt: skip
 
     def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
-        binding_property: Callable[[Json], List[str]] = lambda r: [  # pylint: disable=unnecessary-lambda-assignment
-            ip_config["properties"]["subnet"]["id"]
-            for ip_config in r.get("properties", {}).get("ipConfigurations", [])
-            if "subnet" in ip_config.get("properties", {})
+        first_property: Callable[[AzureNetworkInterface], str] = lambda n: n.id or ""
+
+        second_property: Callable[[AzureNetworkInterface], List[str]] = lambda n: [
```

Comment:
instead of calling those lambdas first and second, consider to use what they give back: `network_id` and `subnet_id` .
same applies to all other first and second properties